patches/chromium/osr_shared_texture_remove_keyed_mutex_on_win_dxgi.patch

```diff
@@ -0,0 +1,100 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Reito <carolwolfking@gmail.com>
+Date: Fri, 19 Apr 2024 11:46:47 +0800
+Subject: Remove DXGI GMB keyed-mutex
+
+This CL is only for reference for a vendor specific change. It is kept for rebasing.
+
+On CEF/electron, FrameSinkVideoCapturer is used in GPU accelerated OSR, and it can be faster
+if we remove the mutex on the shared resource for Windows. This patch is not mandatory, but
+will be nice to be patched for performance gain with no cost.
+
+For reason please see: https://crrev.com/c/5465148
+
+Change-Id: Id90acd587f17acd228ae7cb5ef93005eb8388b80
+
```

Comment:
This description is not helpful as it is currently written.  Please update the description to explain exactly what this patch accomplishes.